runtime/doc/cmdline.txt

```diff
@@ -142,13 +142,15 @@ CTRL-R {register}					*c_CTRL-R* *c_<C-R>*
 		typing CTRL-R and the second character '"' will be displayed
 		to indicate that you are expected to enter the name of a
 		register.
-		The text is inserted as if you typed it, but mappings and
-		abbreviations are not used.  Command-line completion through
-		'wildchar' is not triggered though.  And characters that end
-		the command line are inserted literally (<Esc>, <CR>, <NL>,
-		<C-C>).  A <BS> or CTRL-W could still end the command line
-		though, and remaining characters will then be interpreted in
-		another mode, which might not be what you intended.
+		When used with named registers (A-Z,a-z,0-9) text is inserted
+		literally like pasting with "p". For other registers, the text
+		is inserted as if you typed it, but mappings and abbreviations
```

Comment:
what about clipboard register? that is the main (common) purpose of this change.

Reply:
They are also supported. I guess should mention it in docs.